libraries/chain/include/eosio/chain/config.hpp

```diff
@@ -134,7 +134,7 @@ static_assert(maximum_tracked_dpos_confirmations >= ((max_producers * 2 / 3) + 1
 /**
  * Maximum number of finalizers in the finalizer set
  */
-const static size_t max_finalizers = 64*1024;
+const static size_t max_finalizers = 65535; // largest allowed finalizer policy diff
```

Comment:
I don't think this change is necessary.

Reply:
I think we should change this to 1024 or maybe 512 or even 128.